www/pages/about/goals.md

```diff
@@ -1,3 +1,8 @@
+---
+menu: about
```

Comment:
This is the simplest way to link a page directly with a menu.

Otherwise, you need to create seperate menu files, which we wanted to avoid to begin with, as we already do that.  That would defeat the purpose of compiling those menus at build time.

Reply:
https://github.com/ProjectEvergreen/greenwood/issues/115#issuecomment-522336642  You also suggested this yourself

Reply:
Ah, yeah.  Those weren't meant to part of the implementation / API, just metadata about the page itself that should be passed through (if possible) to the graph when pulling the graph data.